packages/components/src/progress-bar/README.md

```diff
@@ -0,0 +1,47 @@
+# ProgressBar
+
+A simple horizontal progress bar component.
+
+Supports two modes: determinate and indeterminate. A progress bar is determinate when a specific progress value has been specified (from 0 to 100), and indeterminate when a value hasn't been specified.
+
+### Usage
+
+```jsx
+/**
+ * WordPress dependencies
+ */
+import { __experimentalProgressBar as ProgressBar } from '@wordpress/components';
+
+export const MyProgressBar = () => {
+	return <ProgressBar value={ 30 } />;
+};
+```
+
+### Props
+
+The component accepts the following props:
+
+#### `value`: `number`
+
+The progress value, a number from 0 to 100.
+If a `value` is not specified, the progress bar will be considered indeterminate.
+
+-   Required: No
+
+#### `trackColor`: `string`
+
+Optional color of the progress bar track.
+
+-   Required: No
+
+#### `indicatorColor`: `string`
+
+Optional color of the progress bar indicator.
```

Comment:
I think this is a good use case for leveraging the new [`Theme`](https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-theme--default) system rather than adding props to this component. Here is a [list of the CSS variables](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/theme/README.md#writing-themeable-components) that can be used to achieve that.

Reply:
Good call. I'm going to try out the theme system in #53032. Thanks 🙌